spec/ParseFile.spec.js

```diff
@@ -1364,6 +1364,72 @@ describe('Parse.File testing', () => {
       );
     });
 
+    it('works with a period in the file name', async () => {
+      await reconfigureServer({
+        fileUpload: {
+          enableForPublic: true,
+        },
```

Comment:
For the test the file extension should be set explicitly, not to rely on the default option setting:

```suggestion
        fileUpload: {
          enableForPublic: true,
          fileExtensions: ['^[^hH][^tT][^mM][^lL]?$'],
        },
```